composer.json

```diff
@@ -26,7 +26,8 @@
     },
     "require-dev": {
         "symfony/yaml": "~2.3|~3.0",
-        "phpunit/phpunit": "^6.0"
+        "phpunit/phpunit": "^6.0",
+        "phpstan/phpstan": "^0.7"
```

Comment:
I would require it in a different `composer.json` via https://github.com/bamarni/composer-bin-plugin for example to avoid a too grater diff between the deps installed by the consumers and the deps installed for testing